package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "next",
-  "version": "2.0.0-beta.24",
+  "version": "2.0.0-beta.23",
   "description": "Minimalistic framework for server-rendered React applications",
```

Comment:
Haha okay. I'm not a word nazi by any means, but while we're at it, let me just paste this here for future generations.

> There is no such word as minimalistic. The word is minimalist. (Or minimal as Brian points out in the comments). Don't believe me? Try this dictionary.

> There's something a bit ironic about people misusing minimalist by making it longer than necessary ... Although as there are 1.7 million uses of minimalistic on the internet, perhaps ironic doesn't quite cover it.